card/mod/standard/set/self/search.rb

```diff
@@ -1,5 +1,23 @@
 include_set Abstract::SearchParams
 
+def query_args args={}
+  query_args =
+    if wql_keyword? args
+      parse_wql_keyword args
+    else
+      wql_hash.merge args
```

Comment:
`wql_keyword?` doesn't check if the keyword is present it checks if the keyword is a wql expression